packages/plugins/plugin-native/package.json

```diff
@@ -25,6 +25,8 @@
   "dependencies": {
     "@dxos/app-framework": "workspace:*",
     "@dxos/log": "workspace:*",
+    "@dxos/plugin-help": "workspace:*",
+    "@dxos/plugin-navtree": "workspace:*",
```

Comment:
these aren't needed anymore

Reply:
not sure how they got here, prob a bad merge